core/include/moveit/task_constructor/container.h

```diff
@@ -56,7 +56,7 @@ class ContainerBase : public Stage
 
 	/** Callback function type used by traverse functions
 	 *  The callback should return false if traversal should be stopped. */
-	typedef std::function<bool(const Stage&, int depth)> StageCallback;
+	typedef std::function<bool(const Stage&, unsigned int depth)> StageCallback;
```

Comment:
Please update all remaining usages of StageCallback functions as well, e.g.
* [`std::ostream& operator<<(std::ostream& os, const ContainerBase& container)`](https://github.com/ros-planning/moveit_task_constructor/blob/90f5d22e8371b3cd90202114f4afded70b6bc865/core/src/container.cpp#L280)
* [`Introspection::fillTaskDescription`](https://github.com/ros-planning/moveit_task_constructor/blob/90f5d22e8371b3cd90202114f4afded70b6bc865/core/src/introspection.cpp#L207)
* [`Introspection::fillTaskStatistics`](https://github.com/ros-planning/moveit_task_constructor/blob/90f5d22e8371b3cd90202114f4afded70b6bc865/core/src/introspection.cpp#L242)